lib/decanter/core.rb

```diff
@@ -127,16 +127,19 @@ def unhandled_keys(args)
       end
 
       def handled_keys(args)
-        handlers.reduce({}) do |m, h|
-          name, handler = *h
+        handlers.reduce({}) do |acc, curr|
```

Comment:
if we're renaming the arguments can we be even more descriptive?

Reply:
I wanted to 😩 but `handler` was taken. I decided to go for the acc/curr convention instead (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce). Open to other name suggestions tho!